src/main/java/uk/gov/hmcts/ccd/data/casedetails/search/SearchQueryFactoryOperation.java

```diff
@@ -54,8 +55,9 @@ public SearchQueryFactoryOperation(CriterionFactory criterionFactory,
     public Query build(MetaData metadata, Map<String, String> params, boolean isCountQuery) {
         final List<Criterion> criteria = criterionFactory.build(metadata, params);
 
+        Map<String, Object> queryParams = Maps.newHashMap();
         String queryToFormat = isCountQuery ? MAIN_COUNT_QUERY : MAIN_QUERY;
-        String whereClausePart = secure(toClauses(criteria), metadata);
+        String whereClausePart = secure(toClauses(criteria), metadata, queryParams);
```

Comment:
minor. Usage of out parameters is sometimes discouraged. We could have the `secure` method create internally the map of bound parameters and return it together with the whereClausePart. But I appreciate is a bigger change so feel free to ignore.